IrisSoftware/detectEyes.py

```diff
@@ -182,8 +185,23 @@ def takePicture():
     cv2.imwrite("sampleFace.jpg", img)
     camera.release()
 
+def testBlobDetection2():
```

Comment:
Would you be willing to include a brief description of what each test case is doing in the function docstring?

Reply:
Done